frontend/templates/product-list/MetaTags.tsx

```diff
@@ -77,15 +77,15 @@ export function MetaTags({ productList }: MetaTagsProps) {
                <link rel="canonical" href={canonicalUrl} />
                <meta
                   name="description"
-                  content={productList.metaDescription ?? undefined}
+                  content={productList?.overrides?.metaDescription}
                />
             </>
          )}
          <title>{metaTitle}</title>
          <meta property="og:title" content={metaTitle} />
          <meta
             name="og:description"
-            content={productList.metaDescription ?? undefined}
+            content={productList?.overrides?.metaDescription}
```

Comment:
metaDescription, but metaDescription falls  back to description so youre guaranteed to have one because description isn't nullable on the productList type